pkg/download/handler.go

```diff
@@ -44,9 +45,14 @@ func RegisterHandlers(app *buffalo.App, opts *HandlerOpts) {
 	if opts == nil || opts.Protocol == nil || opts.Engine == nil || opts.Logger == nil {
 		panic("absolutely unacceptable handler opts")
 	}
+	noCacheMw := middleware.CacheControl("private, no-store")
```

Comment:
This is a question, should we update to what MDN says about turning off completely?

and have 
```
Cache-Control: no-cache, no-store, must-revalidate
```

Ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control

Reply:
@manugupt1 "no-store" is usually good enough, but I'll keep things consistent with the MDN docs :) -- thanks for the call out!